pkg/utils/nodeclaim/nodeclaim.go

```diff
@@ -74,6 +75,26 @@ func NodeEventHandler(c client.Client) handler.EventHandler {
 	})
 }
 
+func NodeEventHandlerToPods(c client.Client) handler.EventHandler {
```

Comment:
High-level question on the triggering mechanism: Do we need to re-trigger on every pod or just once on the node getting tainted? Does it make sense to put this batch trigger in this existing controller or is this maybe a node controller triggering mechanism?